src/common/oscap_string.h

```diff
@@ -25,8 +25,6 @@
 #include <stdlib.h>
 #include "util.h"
 
-OSCAP_HIDDEN_START;
```

Comment:
errm, what? why?

Reply:
@mpreisler This enables to build SCE in master branch. In current master, SCE doesn't build. SCE needs all functions it uses to be public because SCE is a separate library. I have tried to explain it in commit message of this commit. See https://github.com/OpenSCAP/openscap/pull/890/commits/df50a654a3b66b45ac23e70f223a38da1f7c19de for more details.

Reply:
OK, cool.